client/src/pages/routes/route.ts

```diff
@@ -10,14 +9,14 @@ const menuItems = [
   {
     item: 'Login',
     resource: '/login',
-    canView: null,
+    canView: [AccountType.PET_OWNER, AccountType.PET_SITTER],
```

Comment:
Okay, so if the canView is null it's public and accessible by everyone. We don't want to restrict auth pages to logged in users.